Tutorial split (#145)
pretix/pretix-docs: docs/guides/products/index.md

# Products

> A product is anything sold via pretix: tickets, gift cards, conference t-shirts and so on.

Reviewer: ```suggestion
A product is anything sold via pretix: tickets, gift cards, conference t-shirts, and so on.
```

Normal products are standalone products that can be purchased directly.
Add-on products are products that **cannot** be purchased directly, but only as add-ons to other products during checkout.
The types and numbers of products in the cart decides what add-on products the customer can select.

Cross-selling products are not offered in the shop, but during checkout.
Unlike add-on products, the contents of the cart do not decide what cross-selling products the customer can buy.
Finally, products of the "normal + cross-selling" type are offered as both standalone products and in the cross-selling step.
The "Cross-selling condition" setting below determines how products in categories of the type "cross-selling" and "normal + cross-selling" are offered in your shop.

Click the :btn:Save: button at the bottom of the page.
This takes you back to the product categories page, which now also lists the newly created category.

You can also edit an already existing category and change its name, description, and type, by clicking its name or the :btn-icon:fa3-edit:: edit button next to it in the list.

> 

Reviewer 2: By clicking and dragging the arrow button you can also change the order of the categories in which they are displayed in the back end and online shop, too.

With these two restrictions, customers can only access your event during the time slot for which they bought a ticket.
Once you are happy with your choices, click the :btn:Save: button.

When using pretixSCAN with this setup, the app will ask you to select one of the time slots.
The choice does not matter.
It is only important that you select the check-in list you just set up.
Select any time slot that is part of the event series and then select the check-in list described above.

### Season tickets

Season tickets are often used for public swimming pools, sports clubs, theaters, and other types of venues.
This section explains three methods for setting them up in pretix.
These methods can be used for any ticket that grants access to all events during a certain time period.
It does not matter whether that time period spans several years or just a single day.

The first method, option A, uses memberships.
Option B uses a separate event shop that only sells season tickets.
Option C uses a single season ticket and advanced check-in rules.

> With **Option A**, a customer who purchases a season ticket creates an account with their email address and a password.

Reviewer 2: ```suggestion
With **Option A**, a customer who purchases a season ticket creates a customer account for the organizer with their email address and a password. This customer account is valid for the whole organizer.
```

> That account stores a membership, which acts as the season ticket.

Reviewer 2: One could argument that this could lead to the impression that a membership holds a ticket. Shouldn't we clarify that memberships allow to buy tickets which mandate a special kind of membership?

#### Option A: Memberships and multiple tickets

Option A requires customers to be identified with a customer account.
Thus, you first need to enable the customer accounts feature in your organizer settings in the "Customer accounts" tab.
See also: [Customer accounts](../customer-accounts.md)

![Organizer settings page, on the Customer accounts tab. The boxes next to 'Allow customers to create accounts', 'Allow customers to create accounts', and 'Allow customers to log in with email address and password' have been checked.](../../assets/screens/organizer/customer-accounts.png "Organizer settings–Customer accounts")

Once you have done that, navigate to :navpath:Your organizer → :fa3-user: Customer accounts → Membership types:.
Click the :btn-icon:fa3-plus: Create a new membership type: button.

Give the new membership type a distinct and informative name.
If you want the membership to allow purchase of tickets for multiple different people, check the box next to "Membership is transferable".
If you want the membership to allow purchases for multiple different events or dates with the same start time, check the box next to "Parallel usage is allowed".

> If you want to limit the membership to a certain number of purchases, specify that number in the "Maximum usages" field.

Reviewer 2: 'purchases' means the process of buying tickets and that's not limited. The number of products/tickets one can buy can be limited:

```suggestion
If you want to limit the membership to buy a certain number of products, specify that number in the "Maximum usages" field.
```

Alternatively, if there are long pauses between the individual events throughout the season, you could start with a high price on the season pass and reduce that price manually a few times between events as the season progresses.

With this method, you need to be prepared to check in attendees to two separate events at the same time.
If you are using pretixSCAN, you can use the expert mode to do this.
Open pretixSCAN.
At the top of the screen, tap the name of the event.
Select the individual active event from the list and tap :btn:OK:.
Check the box next to "Expert mode".
Select the check-in list for the event and tap :btn:OK:.

Tap the :btn-icon:fa3-plus:: button.
Select the season passes event and tap :btn:OK:.
Select the check-in list and tap :btn:OK: again.
The screen should now list both your individual event and your season passes event along with the start date, short form, check-in list, and buttons for editing and deleting the entry.

At the top of the screen, tap :btn-icon:fa3-check::.
You should now be able to scan tickets for both events at the same time.

#### Option C: Season access in a single ticket

> Option C is setting up a single ticket that grants access to an event series for an entire season.

Reviewer 2: I would emphasize that this option only works for a single event (series) in contrast to memberships.

```suggestion
Option C is setting up a single ticket that grants access to a single event series for an entire season.
```

 This means that the season pass can only be used once per day.
 - Select "Minutes since last entry" "≤", and "15".
 This means that the season pass can only be used once every fifteen minutes.

![Custom check-in rule page set up according to the instructions above.](../../assets/screens/products/seasontickets_rules1.png "Season tickets rules 1")

The logic is depicted in the screenshot above.
These rules ensure that regular tickets holders are only admitted to their chosen date and if their ticket has not been used before.
They also ensure that season ticket holders are admitted to every date.

### Mixed taxation

The article on [taxes](../taxes.md) has sections explaining how to set up products with mixed taxation using bundles.
The exact method differs depending on whether [tax is included in the price)](../taxes.md#mixed-taxation-tax-included-in-price) or [tax is added on top of the price)](../taxes.md#mixed-taxation-tax-added-on-top-of-price).

## Troubleshooting

### A product does not appear in the ticket shop

If you have created a product and it is not displayed in your ticket shop, perform the following checks:

 1. Check if the product's "active" checkbox is enabled.

>  2. Check if the product's "Available from" or "Available until" settings restrict it to a date range.

Reviewer 2: ```suggestion
 2. Check if the product's "Available from" or "Available until" settings restrict it to a certain date range.
```
